ksql-functional-tests/src/main/java/io/confluent/ksql/test/tools/TestExecutorUtil.java

```diff
@@ -431,26 +443,19 @@ ExecuteResult getExecuteResult() {
     List<DataSource<?>> getSources() {
       return sources;
     }
-
-    public Optional<Long> getWindowSize() {
```

Comment:
Is it safe to remove `windowSize` from these classes? I thought it was needed to support windowed aggregations in the ksql-test-runner tool.

Reply:
It's not being used anymore. It looks like it became obsolete in an earlier patch.